frontend/src/app/shared/utils/lesson.utils.ts

```diff
@@ -22,91 +20,39 @@ export default class Utils {
-            tutorFlagPath: 'assets/lesson-icons/canada-test-flag.svg',
-            tutorName: 'Roger Vaccaro',
-            topics: ['healthy', 'food', 'diet'],
-            subscribersCount: 38,
-            level: 'B1',
-            isDisabled: false,
-        },
-        {
-            id: 3,
-            imgPath: 'assets/lesson-mocks/Photo-2.png',
-            videoId: 'xqAriI87lFU',
-            title: 'How to measure extreme distances',
-            time: '11.00',
-            tutorAvatarPath: 'assets/lesson-mocks/Photo )Patient).png',
-            tutorFlagPath: 'assets/lesson-icons/canada-test-flag.svg',
-            tutorName: 'Roger Vaccaro',
-            topics: ['healthy', 'food', 'diet', 'biology'],
-            subscribersCount: 38,
-            level: 'B1',
-            isDisabled: false,
-        },
-        {
-            id: 4,
-            imgPath: 'assets/lesson-mocks/Photo-3.png',
-            videoId: 'xqAriI87lFU',
-            title: 'What’s so great about the Great Lakes?',
-            time: '11.00',
-            tutorAvatarPath: 'assets/lesson-mocks/Photo )Patient).png',
-            tutorFlagPath: 'assets/lesson-icons/canada-test-flag.svg',
-            tutorName: 'Roger Vaccaro',
-            topics: ['healthy', 'food', 'diet', 'biology'],
-            subscribersCount: 38,
-            level: 'B1',
-            isDisabled: false,
-        },
-        {
-            id: 5,
-            imgPath: 'assets/lesson-mocks/Photo.png',
-            videoId: 'xqAriI87lFU',
-            title: 'No That Easy',
-            time: '11.00',
-            tutorAvatarPath: 'assets/lesson-mocks/Photo )Patient).png',
-            tutorFlagPath: 'assets/lesson-icons/canada-test-flag.svg',
-            tutorName: 'Roger Vaccaro',
-            topics: ['healthy', 'food', 'diet', 'sport', 'biology'],
-            subscribersCount: 38,
-            level: 'B1',
-            isDisabled: false,
-        },
-        {
-            id: 6,
-            imgPath: 'assets/lesson-mocks/Photo-1.png',
-            videoId: 'xqAriI87lFU',
-            title: 'Is It Better to Be Polite or Frank?',
-            time: '11.00',
-            tutorAvatarPath: 'assets/lesson-mocks/Photo )Patient).png',
-            tutorFlagPath: 'assets/lesson-icons/canada-test-flag.svg',
-            tutorName: 'Roger Vaccaro',
-            topics: ['healthy', 'food', 'diet', 'sport', 'biology'],
-            subscribersCount: 38,
-            level: 'B1',
-            isDisabled: false,
-        },
+    static timesList: string[] = [
+        '8.00',
+        '8.30',
+        '9.00',
+        '9.30',
+        '10.00',
+        '10.30',
+        '11.00',
+        '11.30',
+        '12.00',
+        '12.30',
+        '13.00',
+        '13.30',
+        '14.00',
+        '14.30',
+        '15.00',
+        '15.30',
+        '16.00',
+        '16.30',
+        '17.00',
+        '17.30',
+        '18.00',
+        '18.30',
+        '19.00',
+        '19.30',
+        '20.00',
```

Comment:
Can we let user select time? As we will need the whole range + I see no reason why we can't create lesson at 8.15